Cargo.toml

```diff
@@ -32,7 +32,7 @@ keywords = ["flutter", "dart", "ffi", "code-generation", "bindings"]
 categories = ["development-tools::ffi"]
 
 [workspace.dependencies]
-allo-isolate = { version = "0.1.25", features = ["anyhow"] }
+allo-isolate = "0.1.25"
```

Comment:
nit: I guess keeping the original would be good

Reply:
`../flutter_rust_bridge-2.2.0/Cargo.toml`

There is two anyhow feature.

``` toml
[target."cfg(not(target_family = \"wasm\"))".dependencies.allo-isolate]
version = "0.1.25"
features = [
    "anyhow",
    "anyhow",
    "backtrace",
    "zero-copy",
]
```

Reply:
Oh then it is reasonable to remove duplicate